src/styles.scss

```diff
@@ -15,9 +15,13 @@ body {
   overflow-y: scroll;
 }
 
-:root {
-  --default-margin: 20px;
-}
+  :root {
+    --default-margin: 20px;
+  }
+
+  :root {
+    --default-margin2: 20px;
+  }
```

Comment:
Men er å huske at dette var bare for å teste? Om så bare fjern om det ikke skal brukes